src/init.js

```diff
@@ -29,13 +29,21 @@ export function initApp (queryString = window.location.search) {
   if (queryString.length === 0) {
     updateURL(mapView)
   }
+
   // Initializing lat/lng and zoom
   store.dispatch(recenterMap(coordinates, mapView.zoom))
   // Initializing map search bar
   store.dispatch(setLocation(coordinates, label))
   // Initializing dates
   store.dispatch(setDate(date.startDate, date.endDate))
 
+  // Initializing analysis view name
+  if (object.viewName) {
+    const viewName = object.viewName
+    document.title = viewName + ' | OpenTraffic Analyst UI '
```

Comment:
What about when the link is shared? If the link is shared, then if there is a viewName the document title should be changed. Should I set the document.title in the component constructor as well as handleSubmit()? Otherwise when the link is shared, the document.title does not change until the input is submitted/changed.